html/partials/services.html.haml

```diff
@@ -0,0 +1,24 @@
+- if services
+  %div
+    %a.scope_anchor{:id => "services"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace